.github/workflows/update-semconv-integration-branch.yml

```
@@ -90,6 +90,7 @@ jobs:
         with:
           node-version-file: .nvmrc
           cache: npm
+          cache-dependency-path: tmp/package-ci.json
```

We don't create a temporary package spec so let's use the main one:

```suggestion
          cache-dependency-path: package.json
```

---

Or we could just turn off caching?